### PR TITLE
Add Volume Controls to Media Control Card

### PR DESCRIPTION
**Pull request by timmo001**
## Proposed change

Adds volume controls for those that support control.

Screenshot_20200321_141944

## Type of change

- [x] New feature (thank you!)

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

### CODE REVIEW

File: src/panels/lovelace/cards/hui-media-control-card.ts [hunk 1]
```diff
@@ -505,40 +506,28 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
...
-        : undefined;
+    if (state === "on" && supportsFeature(stateObj, SUPPORT_TURN_OFF)) {
+      buttons.push({
+        icon: "hass:power",
+        action: "turn_off",
+      });
     }
 
-    const buttons: ControlButton[] = [];
+    if (state === "idle" && supportsFeature(stateObj, SUPPORTS_PLAY)) {
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-media-control-card.ts]: We will now add next and previous buttons in idle mode, we should not.

**Comment by bramkragten** [comment]
I don't think this is a great solution, I think the volume buttons are too big. 
I would prefer a slider in a popup when you hover a volume button or something.

File: src/panels/lovelace/cards/hui-media-control-card.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/lovelace/cards/hui-media-control-card.ts]: image

**Comment by timmo001** [comment]
OK. This needs additional thought so I'll close up for now and reopen if I can think of a solution.

Needs to be something that is touch and desktop friendly

**Pull request closed without merging**